.github/workflows/conformance-ipsec-e2e.yaml

```diff
@@ -243,7 +243,7 @@ jobs:
         with:
           job-name: conformance-ipsec-e2e-key-rotation-${{ matrix.name }}
           operation-cmd: |
-            KEYID=$(kubectl get secret -n kube-system cilium-ipsec-keys -o go-template --template={{.data.keys}} | base64 -d | cut -c 1)
```

Comment:
The same issue exists in the documentation: https://docs.cilium.io/en/latest/security/network/encryption-ipsec/#key-rotation.

(Sorry, just saw that while working on my own update of that doc.)

Reply:
Good catch! Let me file a separate PR.

Reply:
https://github.com/cilium/cilium/pull/30000